styles.css

```diff
@@ -0,0 +1,295 @@
+  width: 30px;
+  padding-top: 36px;
+}
+
+.burger__menu {
+  width: auto;
+  height: auto;
+  cursor: pointer;
+  background-color: #fff;
+  border: none;
+  transition: opacity .5s;
+}
+
+.burger__menu:hover {
+  opacity: .5;
+}
+
+.menu {
+  display: inline-block;
+  display: flex;
+  align-items: center;
+}
+
+.menu__item {
+  position: relative;
+  margin-right: 26px;
+  font: 500 18px/21px Roboto,arial,helvetica,sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s;
+}
+
+.menu__item:last-child {
+  margin-right: 0;
+}
+
+.menu__item:after {
+  position: absolute;
+  left: 0;
+  display: block;
+  content: "";
+  width: 100%;
+  height: 3px;
+  background: #0056ec;
+  opacity: 0;
+  transition: opacity .5s;
+}
+
+.menu__item:hover {
+  opacity: .5;
+}
+
+.menu__item:hover::after, .menu__item_active:after {
+  opacity: 1;
+}
+
+.main {
+  padding: 97px 65% 160px 0;
+  background: top right no-repeat url(img/Frame.svg);
+}
+
+.main__title {
+  margin: 0 0 40px 0;
+  font: bold 66px/87px 'PT Serif',serif;
+}
+
+.main__subtitle {
+  margin: 0 0 41px 0;
+  font: 22px/25px Roboto,arial,helvetica,sans-serif;
+}
+
+.main__title-text_highlight {
+  background: #eCf3ff;
+}
+
+.main__button {
+  display: inline-block;
+  font: 18px/21px Roboto,arial,helvetica,sans-serif;
+  color: #fff;
+  background: #0056ec;
+  padding: 18px 30px;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: opacity .5s;
+}
+
+.main__button:hover {
+  opacity: .5;
+}
+
+@media (max-width: 1366px) {
+ .wrapper {
+   margin-left: 120px;
+   width: 70%;
+ }
+
+ .header {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 30px;
+ }
+
+ .menu {
+   display: flex;
+   align-items: center;
+ }
+
+ .menu__item {
+   position: relative;
+   margin-right: 26px;
+   font: 500 14px/16px Roboto,arial,helvetica,sans-serif;
+   color: #000;
+   text-decoration: none;
+   transition: opacity .5s;
+ }
+
+ .main {
+   padding: 0 58% 160px 0;
+   background: top right no-repeat url(img/Frame.svg);
+   background-size: 54%;
+   width: 390px;
+ }
+
+ .main__title {
+   margin: 0 0 40px 0;
+   font: bold 50px/66px 'PT Serif',serif;
+ }
+
+ .main__subtitle {
+   margin: 0 0 41px 0;
+   font: 500 18px/21px Roboto,arial,helvetica,sans-serif;
+ }
+
+ .main__button {
+   font: 500 14px/16px Roboto,arial,helvetica,sans-serif;
+ }
+}
+@media (max-width: 1170px){
+ .wrapper {
+   margin-left: 40px;
+   width: 78%;
+ }
+
+
+ .main {
+   padding: 0 57% 160px 0;
+   background: top right no-repeat url(img/Frame.svg);
+   background-size: 55%;
+   width: 390px;
+ }
+
+ .main__button {
+    margin-bottom: 140px;
+    font: 500 14px/16px Roboto,arial,helvetica,sans-serif;
+ }
+}
+
+@media (max-width: 768px) {
+  .wrapper {
+    width: 89%;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .logo {
+    visibility: hidden;
+     padding: 53px 0 0 0;
+   }
+
+   .logo__circle_color_blue, .logo__cirle_color_dark-blue {
+     visibility: visible;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/site99/pullRequest?prid=6205931)